scrum.py

```diff
@@ -112,14 +146,14 @@ def get_story_label_ids(self, user_story_model, all_board_labels):
         return story_label_ids
 
 
-    def get_all_board_members(self):
+    def get_all_board_members(self, config):
         url = "https://api.trello.com/1/boards/627c210aa0ed4a48c3dd069c/members"
```

Comment:
Do not forget to replace this hardcoded `627c210aa0ed4a48c3dd069c` id

Reply:
fixed